packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -276,21 +229,29 @@ const props = defineProps<{
 const emit = defineEmits(['update-project']);
 
 const tabStore = useTabStore();
-const router = useRouter();
-const resources = useResourcesStore();
 
-const newModelId = ref<string>('');
-const isNewModel = ref<boolean>(true);
+// Makes asset from workflow node appear
+const openedWorkflowNodeStore = useOpenedWorkflowNodeStore();
+const workflowNodeAssetId = computed(() => {
+	let index: number = 0;
+
+	if (openedWorkflowNodeStore?.workflowNode?.outputs.length) {
+		index = openedWorkflowNodeStore.workflowNode.outputs.length - 2;
```

Comment:
Why `-2`? If length is 1, won't index be -1?

Reply:
Once the model is ran a new output port is added. That new port is empty though so I am just grabbing the one before that. I'll write this better though as I haven't accounted for one empty output port.